arch/riscv/crc32_riscv.c

```diff
@@ -0,0 +1,98 @@
+/* crc32_rvv.c - RISCV Zbc version of crc32
```

Comment:
I think this file should be named crc32_zbc.c. Because riscv is already in the folder name so it's a bit redundant.